File: easy/DeleteColumnstoMakeSorted[944]/DeleteColumnstoMakeSorted.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        count = 0
        for i in range(len(strs[0])):
            prev = ""
            for idx in range(len(strs)):
                curr = strs[idx][i]
                if curr < prev:
                    count += 1
                    break
                prev = curr
        return count

File: easy/DeleteColumnstoMakeSorted[944]/DeleteColumnstoMakeSorted.rs
// optimal
impl Solution {
    pub fn min_deletion_size(strs: Vec<String>) -> i32 {
        let mut count = 0;
        for i in 0..strs[0].len() {
            for idx in 0..strs.len() - 1 {
                if &strs[idx][i..i + 1] > &strs[idx + 1][i..i + 1] {
                    count += 1;
                    break;
                }
            }
        }
        count
    }
}
